service/service.go

```diff
@@ -47,6 +47,10 @@ func Run(ctx context.Context, serviceList *ExternalServiceList, buildTime, gitCo
 	cantabularClient := serviceList.GetCantabularClient(ctx, cfg)
 	datasetAPIClient := serviceList.GetDatasetAPIClient(ctx, cfg)
 	s3Client, err := serviceList.GetS3Client(ctx, cfg)
+	if err != nil {
+		log.Event(ctx, "failed to initialise s3 client", log.FATAL, log.Error(err))
```

Comment:
Instead of logging and then returning the error, can we return the error wrapped with it's context?
`return nil, fmt.Errorf("failed to initialise s3 client: %w", err)